internal/models/integration.go

```diff
@@ -80,9 +94,59 @@ func ConvertToIntegrationModel(ctx context.Context, integration *apono.Integrati
 		}
 	}
 
+	data.ResourceOwnerMappings = ConvertResourceOwnersMappingToModel(integration.ResourceOwnersMappings)
+	data.IntegrationOwners = ConvertIntegrationOwnerToData(integration.IntegrationOwners)
+
 	return &data, nil
 }
 
+func ConvertIntegrationOwnerToData(Owners []aponoapi.IntegrationOwnerTerraform) []IntegrationOwner {
+	if Owners == nil {
+		return nil
+	}
+	var integrationOwners = make([]IntegrationOwner, len(Owners))
+	for i, in := range Owners {
+		var AttributeValue = make([]types.String, len(in.AttributeValue))
+		for j, av := range in.AttributeValue {
+			AttributeValue[j] = basetypes.NewStringValue(av)
+		}
+		var IntegrationId types.String
+		if in.IntegrationId.IsSet() {
+			IntegrationId = basetypes.NewStringValue(*in.IntegrationId.Get())
+		} else {
+			IntegrationId = basetypes.NewStringNull()
+		}
```

Comment:
```suggestion
		IntegrationId := basetypes.NewStringPointerValue(in.IntegrationId.Get())
```
this code can be simplify with using the pointer to init the variable